scripts/py_matter_idl/matter_idl/matter_grammar.lark

```diff
@@ -2,6 +2,13 @@ struct: struct_qualities "struct"i id "{" (struct_field ";")* "}"
 struct_quality: "fabric_scoped"i -> struct_fabric_scoped
 struct_qualities: struct_quality*
 
+// Composing elements may be at different maturity level
+// This only contains non-stable items. Items without a maturity
+// flag are to be assumed "stable"
```

Comment:
It should be optionally possible to explicitly state "stable"

Reply:
Added as a supported keyword. 

I would however generally discourage its use.